templates/home/templates/product-is-template.html.twig

```diff
@@ -1,7 +1,7 @@
 <section class="py-5">
     <div class="container">
         <h2 class="h2 text-primary text-uppercase text-center">{{ section_title| trans }}</h2>
-        <div class="row pt-3 pt-md-4 pb-3">
+        <div class="row justify-content-between pt-3 pt-md-4 pb-3">
```

Comment:
between or center ? You can ask PO about it
<img width="1440" alt="image" src="https://github.com/user-attachments/assets/d9ca6bd2-9914-45a7-b11a-7ac5a6fbf573" />

<img width="1440" alt="image" src="https://github.com/user-attachments/assets/695b3e5e-439b-40a5-8132-130e038d9dea" />